src/auth/basic/User.cc

```diff
@@ -25,14 +25,14 @@ Auth::Basic::User::~User()
     safe_free(passwd);
 }
 
-int32_t
+Auth::Ttl
 Auth::Basic::User::ttl() const
 {
     if (credentials() != Auth::Ok && credentials() != Auth::Pending)
-        return -1; // TTL is obsolete NOW.
+        return Auth::User::ttl(); // TTL is obsolete NOW.
```

Comment:
Here and and in similar contexts, please either continue to return `-1` or introduce a named Auth::Ttl constant and return that. Either would be more readable and safe than the proposed "use TTL computed by Auth::User::ttl() while assuming that that value is a negative TTL" change.